examples/grid_experiment/grid.cc

```diff
@@ -0,0 +1,73 @@
+#include "common/map_coordinate.h"
+#include <iostream>
+#include <string>
+#include <cstring>
+#include "third_party/UTM.h"
+#include <iomanip>
+
+using units::length::meter_t;
+using units::angle::degree_t;
+
+using BoBRobotics::MapCoordinate::UTMCoordinate;
+using BoBRobotics::MapCoordinate::LatLon;
+using BoBRobotics::MapCoordinate::WGS84;
+using BoBRobotics::MapCoordinate::Ellipsoid;
+using BoBRobotics::MapCoordinate::Cartesian;
+using BoBRobotics::MapCoordinate::latLonToCartesian;
+using BoBRobotics::MapCoordinate::cartesianToLatLon;
+using BoBRobotics::MapCoordinate::shiftLatLon;
+using BoBRobotics::MapCoordinate::shiftUTM;
+
+using UTM::LLtoUTM;
+using UTM::UTMtoLL;
+using CARCoordiante = Cartesian<WGS84>;
+using GPSCoordinate = LatLon<WGS84>;
+
+int main()
+{
+    // test for GPS Coordinate by moving it 10 meters in x direction
+    double SHIFT = 30;
+
+    GPSCoordinate home;
+    home.lat = degree_t(50.87002350275288);
+    home.lon = degree_t(0.0018240860639551215);
```

Comment:
If you put ``using namespace units::literals;`` at the top of this function you can then specify literals in degrees like ``50.87002350275288_deg``